tests/snuba/api/endpoints/test_organization_events_trace.py

```diff
@@ -0,0 +1,312 @@
+from uuid import uuid4
+
+from django.core.urlresolvers import reverse
+
+from sentry.utils.samples import load_data
+from sentry.testutils import APITestCase, SnubaTestCase
+from sentry.testutils.helpers.datetime import before_now, iso_format
+
+
+class OrganizationEventsTrendsLightEndpointTest(APITestCase, SnubaTestCase):
+    FEATURES = ["organizations:trace-view-quick", "organizations:global-views"]
+
+    def create_event(self, trace, spans, parent_span_id, project_id):
+        data = load_data(
+            "transaction",
+            trace=trace,
+            spans=spans,
+        )
+        data["timestamp"] = iso_format(before_now(minutes=1))
+        data["start_timestamp"] = iso_format(before_now(minutes=5))
```

Comment:
nit: You can add both `timestamp` and `start_timestamp` to `load_data` instead of manually setting it here.